json/upickle/src/main/scala/tapir/upickle/TapirJsonuPickle.scala

```diff
@@ -0,0 +1,28 @@
+package tapir.json.upickle
+
+import java.nio.charset.StandardCharsets
+import scala.util.{Try, Success, Failure}
+import tapir.Codec.JsonCodec
+import tapir.DecodeResult.{Error, Value}
+import tapir._
+import upickle.default.{ReadWriter => RW}
+import upickle.default.{read, write}
+
+trait TapirJsonuPickle {
+
+  implicit def encoderDecoderCodec[T: RW: SchemaFor, M]: JsonCodec[T] = new JsonCodec[T] {
```

Comment:
What is the type parameter `M` for?

Reply:
Removed. That must have been from an earlier iteration of the code :)